kubernetes/aks/system/vault/namespace.yml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vault
+  labels:
+    name: vault
```

Comment:
The namespace of vault should probably be in common (if so move the namespace for the GKE cluster inside common).